src/info/mod.rs

```diff
@@ -304,27 +304,58 @@ impl Info {
     }
 
     fn get_language_field(&self, title: &str) -> String {
-        let mut language_field = String::from("");
-
+        let language_bar_length = 26;
```

Comment:
Based on the screenshot, this seems to line up nicely. If this was 29 characters, it might be more symmetrical with the ASCII art, which maxes out at 40 characters wide. (29 + `"Languages: ".len()`).

Reply:
If the language bar length could be fixed to 29, this would indeed make sense. However with the possible extra characters, this value can go up to 35, and if we add `"Languages: ".len()` up to 46.